new-findings.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+GO_SRC_PATH="$(go env GOPATH)/src"
+CURRENT_BRANCH=$(git branch --show-current)
+
+if [[ ${CURRENT_BRANCH} == "master" ]]; then
+    echo  "Please run from feature branch"
+    exit 1
+fi
+
+echo "Switching to master"
+
+git checkout master
+
+echo "Installing levee from master"
+
+cd cmd/levee
+
+go install
+
+echo "Running levee(master) against k8s"
+
+cd $GO_SRC_PATH/k8s.io/kubernetes
+
+CFG_PATH="$(realpath ./hack/testdata/levee/levee-config.yaml)"
+
+make clean
```

Comment:
Do you have evidence that `make clean` is necessary?  I would think / hope that it isn't, and it will bloat your runtime.  Ditto below.

Reply:
Yup, it is if you don't do that before each `make vet ...` there is no output.

Reply:
I'm not convinced that it will *systematically* mess up the following `make vet`.

The impact on runtime should be minimal though.